doc/repository-spec-v2.md

```diff
@@ -64,6 +64,9 @@ repository.
 
 ### Inspect a specific version of a package
 
+**Deprecated** Use `<PUB_HOSTED_URL>/api/packages/<PACKAGE>` instead and find
```

Comment:
I think it's enough to say:
```md
**Deprecated** as of Dart 2.8, use "List all versions of a package" instead.
```
In particular, we'll want to know what Dart version we stopped using this in. Such that when we decide to remove the end-point we know how many Dart versions will be affected.

I assume it'll be years before we remove the end-point, and we might have forgotten in the mean time.

Reply:
Done